### ISSUE TITLE
Forbid subclasses to have the exact same methods as superclasses

### ISSUE THREAD

**Issue by sobolevn** [PyCQA/flake8-pyi#159]
Right now this code is not an error:

```python
class A:
    def some(self) -> int: ...

class B(A):
    def some(self) -> int: ...
```

But, it is not a good style. I propose to add a check for this.
Related https://github.com/python/typeshed/pull/7073

**Comment by AlexWaygood** [comment]
I agree that this would be great. It might be tricky to implement (but my mind is already racing with ideas about how to do it 😀).

We would also have to exclude dunder methods, because we would get a lot of false positives (see discussion in https://github.com/python/typeshed/pull/6877).

**Comment by sobolevn** [comment]
Offtopic, @AlexWaygood if you are intersted in `flake8` plugins, I welcome you giving https://github.com/wemake-services/wemake-python-styleguide a try. It has lots of open feature requests (all are pretty non-trivial).

**Comment by AlexWaygood** [comment]
> Offtopic, @AlexWaygood if you are intersted in `flake8` plugins, I welcome you giving https://github.com/wemake-services/wemake-python-styleguide a try. It has lots of open feature requests (all are pretty non-trivial).

I'll add it to my to-do list 😄

**Comment by AlexWaygood** [comment]
As discussed in https://github.com/python/mypy/issues/13321, this check would be a better fit for a linter that can harness the power of a type checker; let's do that instead.
